en-US/download_button.lang

```diff
@@ -46,6 +50,14 @@ Get it free from the App Store
 Firefox Beta
 
 
+;<span>Firefox</span> Developer Edition
```

Comment:
In the past we decided to keep this string out of the file on purpose. There's no "for" to localize, and the name is not supposed to be localized. 

I'm OK with exposing "Firefox Nightly", I would leave DevEdition out.